api/types/plugin.go

```diff
@@ -147,14 +147,22 @@ func (p *PluginV1) CheckAndSetDefaults() error {
 			return trace.BadParameter("labels must be specified")
 		}
 	case *PluginSpecV1_Jamf:
-		if settings.Jamf.JamfSpec.ApiEndpoint == "" {
-			return trace.BadParameter("api endpoint must be set")
+		// Check Jamf settings.
+		if settings.Jamf != nil {
+			return trace.BadParameter("missing Jamf settings")
+		}
```

Comment:
I think this is meant to be

```go
if settings.Jamf == nil {
    return trace.BadParameter("missing Jamf settings")
}
```

Reply:
Oops! fix in https://github.com/gravitational/teleport/pull/27814. 
Many thanks!